python/ray/autoscaler/_private/load_metrics.py

```diff
@@ -138,23 +140,21 @@ def prune_active_ips(self, active_ips):
         active_ips.add(self.local_ip)
 
         def prune(mapping, should_log):
-            unwanted = set(mapping) - active_ips
-            for unwanted_key in unwanted:
+            unwanted_ips = set(mapping) - active_ips
+            for unwanted_ip in unwanted_ips:
                 if should_log:
-                    logger.info("LoadMetrics: "
-                                "Removed mapping: {} - {}".format(
-                                    unwanted_key, mapping[unwanted_key]))
-                del mapping[unwanted_key]
-            if unwanted and should_log:
-                # TODO (Alex): Change this back to info after #12138.
+                    logger.info("LoadMetrics: " "Removed ip: {unwanted_ip}.")
```

Comment:
Logging the value produced a confusing log message.

Reply:
Maybe add a reason (why removed) to the message?

Reply:
great idea -- I think whoever calls this logs that the ips are `stale` but doesn't explain what that means